test/unit/attributes.js

```diff
@@ -4,127 +4,306 @@ import helpers from '../lib/helpers';
 import skate from '../../src/skate';
 
 describe('Attribute listeners', function () {
-  it('should listen to changes in specified attributes', function (done) {
+  'use strict';
+
+  describe('default values', function () {
+    it('should set a default value using the "default" option', function () {
+      var MyEl = skate('my-el', {
```

Comment:
You'll need to update these tests to use `safeTagName('my-el')` to avoid re-registering the same tag name.